xmodule/library_content_block.py

```diff
@@ -97,23 +104,31 @@ class LibraryContentBlock(
 
     show_in_read_only_mode = True
 
-    # noinspection PyMethodParameters
-    @classproperty
-    def completion_mode(cls):  # pylint: disable=no-self-argument
-        """
-        Allow overriding the completion mode with a feature flag.
-
-        This is a property, so it can be dynamically overridden in tests, as it is not evaluated at runtime.
-        """
-        if settings.FEATURES.get('MARK_LIBRARY_CONTENT_BLOCK_COMPLETE_ON_VIEW', False):
-            return XBlockCompletionMode.COMPLETABLE
+    resources_dir = 'assets/library_content'
 
-        return XBlockCompletionMode.AGGREGATOR
+    preview_view_js = {
+        'js': [],
+        'xmodule_js': resource_string(__name__, 'js/src/xmodule.js'),
+    }
+    preview_view_css = {
+        'scss': [],
+    }
+    mako_template = 'widgets/metadata-edit.html'
+    studio_js_module_name = "VerticalDescriptor"
+    studio_view_js = {
+        'js': [
+            resource_string(__name__, 'js/src/vertical/edit.js'),
+        ],
+        'xmodule_js': resource_string(__name__, 'js/src/xmodule.js'),
+    }
+    studio_view_css = {
+        'scss': [],
+    }
```

Comment:
```suggestion
    mako_template = 'widgets/metadata-edit.html'
```
This is all old stuff from before the xmodule asset refactoring.